typescript/src/enums.ts

```diff
@@ -244,3 +244,45 @@ export enum PURCHASES_ARE_COMPLETED_BY {
    */
   REVENUECAT = "REVENUECAT",
 }
+
+/**
+ * Configuration option that specifies that your app will complete purchases.
+ * @public
+ */
+export type PURCHASES_ARE_COMPLETED_BY_MY_APP = {
```

Comment:
Hmm I’m honestly not sure about naming this uppercase. Feels like it should be an enum. Probably better to do: `PurchasesAreCompletedByMyApp` for the type?

Reply:
Works for me, I'll make the change!